plugins/europeana/labeling.js

```diff
@@ -0,0 +1,40 @@
+const fieldLabels = {
+  default: {
+    dcContributor: 'Contributors',
+    dcCreator: 'Creators',
+    dcDescription: 'Description',
+    dcTitle: 'Title',
+    dcType: 'Type of object',
+    dctermsCreated: 'Creation date',
+    edmCountry: 'Providing country',
+    edmDataProvider: 'Providing institution',
+    edmRights: 'License of the media in this record (unless otherwise specified)'
+  },
+  webResource: {
+    edmRights: 'License for this media resource'
+  }
+};
+
+function lookupFromContext(key, context = 'default') {
+  context = fieldLabels.hasOwnProperty(context) ? fieldLabels[context] : fieldLabels['default'];
```

Comment:
Perhaps define a new `const` here named `contextLabels` instead of overwriting the parameter.

Reply:
If `context` parameter has been passed and does not exist as a property of `fieldLabels`, this method should throw an error instead of falling back to default as the caller has explicitly requested an unavailable context.

Reply:
see my comment above about not explicitly wanting to tie contexts to labelling.